src/command.cpp

```diff
@@ -13,6 +13,10 @@
 #include "settingsholder.h"
 #include "simplenetworkmanager.h"
 
+#ifdef MVPN_WINDOWS
+    #include "Windows.h"
```

Comment:
Winging it a bit here so please advise! Happy to make changes obviously if there is a better/snazzier/more correct way to do this thing.